lib/at/locker.toit

```diff
@@ -7,6 +7,8 @@ import .session
 /**
 The locker can wrap a $Session, and ensure both exclusive access to the
   Session and a queue in case of concurrent requests.
+
+Deprecated.
```

Comment:
Provide information on what should/can be used instead. Same for the others.

Reply:
There are no obvious alternatives available.